dietpi/dietpi-software

```diff
@@ -462,6 +464,7 @@ _EOF_
 		EMONHUB=$(cat "$FP_INSTALLED_FILE" | grep -m1 'EMONHUB ' | awk '{print $2}')
 		EMONCMS=$(cat "$FP_INSTALLED_FILE" | grep -m1 'EMONCMS ' | awk '{print $2}')
 		RPIMONITOR=$(cat "$FP_INSTALLED_FILE" | grep -m1 'RPIMONITOR ' | awk '{print $2}')
+		NETDATA=$(sed -n 's/^NETDATA\s*\(.*\)/\1/p' "$FP_INSTALLED_FILE")
```

Comment:
Need to add a space after ```s/^NETDATA```. This prevents items with NETDATA at the start from being picked up in the file.

EG:
NETDATA
NETDATA_SOMEADDITION
Will both be picked up without the space at the end.